datasetinsights/stats/visualization/bbox2d_plot.py

```diff
@@ -158,5 +161,44 @@ def _add_single_bbox_on_image(
         rec_right_bottom = (rectangle_right, rectangle_bottom)
 
         _cv2.rectangle(image, rec_left_top, rec_right_bottom, color, -1)
+        _add_label_on_image(
```

Comment:
This method does not "add label" instead it tries to do label cropping. Since you have a method called `_add_label_on_image`, why not just move everything from line 135 to 163 to this method? I would expect to see this 
```
if label:
   _add_label_on_image(...)

def _add_label_on_image(...):
   ...
   _cv2.rectangle(...)
   _ fix_labe_at_image_edge(...)

def _fix_label_at_image_edge(...)
   ...
```

Reply:
Fixed.